docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -472,6 +472,17 @@ When moving modules between collections
 
 See :ref:`Migrating content to a different collection <migrate_to_collection>` for complete details.
 
+Generally we do not object to moving content between collections, or moving content from collections included in Ansible to collections outside the Ansible package, as long as semantic versioning is not violated. More precisely, replacing content by redirects is only a minor change if the destination collection is the dependency of the source collection of the move. (See :ref:`coll_dependencies` for adding new dependencies to collections included in Ansible.)
+
+For the community "catch all" collections, we have slightly different rules. We allow to move content out of community.general and community.network to other collections outside of Ansible under the following conditions:
```

Comment:
Should we really mention `community.network` here [now that we've decided to deprecate and remove it](https://forum.ansible.com/t/8030/35)?

Reply:
The collection is still alive until in ~6 months, since until then it will appear in Ansible releases. Also it's still possible that someone revives it. I would keep it in here, it's easy to remove it from here later on.

Reply:
Sounds good to me.